README.md

```diff
@@ -1,7 +1,15 @@
 # Olin.js 2015
 
-To generate `index.html`:
+## Dependencies
+Ensure that jade is installed (`npm install -g jade`).
```

Comment:
Might want to mention that Python `requests` is also a dependency (just ran into that trying to test the build system).